testeng/jobs/oep2Report.groovy

```diff
@@ -0,0 +1,56 @@
+package testeng
+
+job('oep2-report') {
+
+    description('Generate a report describing OEP-2 compliance accross edX repos')
+    authorization {
+        blocksInheritance(true)
+        permissionAll('edx')
+        permission('hudson.model.Item.Discover', 'anonymous')
+    }
+    concurrentBuild(false)
+    label('jenkins-worker')
+    scm {
+        git {
+            remote {
+                url('https://github.com/edx/repo-tools')
+            }
+            branch('*/master')
+            browser()
+        }
+    }
+    triggers {
+        cron('@midnight')
+    }
+    wrappers {
+        timestamps()
+        sshAgent('jenkins')
+        credentialsBinding {
+           file('OEP_REPORT_TOKEN', 'GITHUB_STATUS_OAUTH_TOKEN')
+        }
+    }
+    steps {
+        virtualenv {
+            name('oep-venv')
+            nature('shell')
+            clear(true)
+            command(readFileFromWorkspace('testeng/resources/create-oep-report.sh'))
+        }
+    }
+    publishers {
+        archiveJunit('oep2-report.xml') {
+            retainLongStdout(false)
+            healthScaleFactor(1.0)
```

Comment:
Trying to seed this gave me errors on test-jenkins as well as a local instance created from our jenkins-configuration repo. It looks like `healthScaleFactor()` becomes available in version 1.52 of the job-dsl plugin, and we are currently on 1.45